# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pulseaudio/hotfix-arm-neon.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- pulseaudio-16.1/src/pulsecore/meson.build.vanilla	2023-05-15 12:35:23.280104212 +0200
+++ pulseaudio-16.1/src/pulsecore/meson.build	2023-05-15 12:42:53.757136738 +0200
@@ -177,14 +177,14 @@
 simd_variants = [
   { 'mmx' : ['remap_mmx.c', 'svolume_mmx.c'] },
   { 'sse' : ['remap_sse.c', 'sconv_sse.c', 'svolume_sse.c'] },
-  { 'neon' : ['remap_neon.c', 'sconv_neon.c', 'mix_neon.c'] },
+  #{ 'neon' : ['remap_neon.c', 'sconv_neon.c', 'mix_neon.c'] },
 ]
 
 libpulsecore_simd_lib = []
 
 foreach simd_kwargs : simd_variants
 
-  if host_machine.cpu_family() == 'arm' and 'neon' in simd_kwargs
+  if 'neon' in simd_kwargs
     if not cc.compiles('''
         #include <arm_neon.h>
         int main() {
